src/VirtoCommerce.Platform.Web/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -1,71 +1,32 @@
-using System;
-using System.Linq;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
-using VirtoCommerce.Platform.Core.Common;
-using VirtoCommerce.Platform.Core.Modularity;
-using VirtoCommerce.Platform.Modules.External;
-using VirtoCommerce.Platform.Web.Infrastructure;
 
 namespace VirtoCommerce.Platform.Modules
 {
     public static class ServiceCollectionExtensions
     {
-        public static IServiceCollection AddModules(this IServiceCollection services, IMvcBuilder mvcBuilder, Action<LocalStorageModuleCatalogOptions> setupAction = null)
+        public static IServiceCollection ConfigureServer(this IServiceCollection services)
```

Comment:
Maybe better to call `ConfigureWebServer`

Reply:
agree